rpcs3/rpcs3qt/save_data_list_dialog.cpp

```diff
@@ -158,10 +87,19 @@ save_data_list_dialog::save_data_list_dialog(const std::vector<SaveDataEntry>& e
 	LoadEntries();
 	UpdateList();
 
+	connect(m_list, &QTableWidget::cellChanged, [&](int row, int col) {
```

Comment:
this whole connect looks a bit bloated to me. too much extra one time use variables (just to annoy you with extra stuff)